apps/evm/src/clients/api/queries/getVoterHistory/types.ts

```diff
@@ -3,7 +3,7 @@ import type { Address, ByteArray } from 'viem';
 
 export interface ProposalActionApiResponse {
   actionIndex: number;
-  calldata: `0x${string}` | ByteArray;
+  calldata: Address | ByteArray;
```

Comment:
Hmm, I feel this is better typed as `Hex`. Both are `0x${string}` at the end of the day, but this isn't really an `Address`.

Reply:
Good point.